poc/flp_generic.py

```diff
@@ -888,3 +918,82 @@ def test_vec_set_type_param(self, test_vec):
         test_vec['bits'] = self.bits
         test_vec['chunk_length'] = self.chunk_length
         return ['length', 'bits', 'chunk_length']
+
+
+class Sum2(Valid):
```

Comment:
Can this be more descriptive?

Reply:
Renamed to `SumOfRangeCheckedInputs`. This is probably overly verbose; we may consider renaming it to `Sum` as part of #306.